mods/ra/tilesets/desert.yaml

```diff
@@ -2718,28 +2676,70 @@ Templates:
 			2: Water
 			3: Beach
 			5: Beach
-	Template@439:
-		Id: 439
-		Images: sh62.des
-		Size: 6,1
+	Template@441:
+		Id: 441
+		Images: sh64.des
+		Size: 2,3
+		Categories: Beach
+		Tiles:
+			0: River
+			1: Rough
+			2: River
+			3: River
+			4: River
+			5: Rock 
```

Comment:
One more trailing space found. You can use `git commit --amend`  to attach the change to the last commit, so you don't need create a new commit and squash it again, after that I would approve this PR.